modules/grafana/manifests/dashboards/application_dashboard.pp

```diff
@@ -105,6 +106,17 @@
     $elasticsearch_stats = []
   }
 
+  if $show_external_request_time {
+    $external_request_row = [
+      [
+        'content_store_request_time',
+        'registry_request_time',
```

Comment:
I know what the Content Store and Rummager are, but what's the registry?

Reply:
It’s a cache of information that we need to be able to provide filtering for our finders.